pkg/terra/relay.go

```diff
@@ -24,50 +26,49 @@ type Logger interface {
 	Fatalf(format string, values ...interface{})
 }
 
-type TransmissionSigner interface {
-	Sign(msg []byte) ([]byte, error)
-	PublicKey() cryptotypes.PubKey
+type MsgEnqueuer interface {
+	Enqueue(contractID string, msg []byte) (int64, error)
+	Start() error
+	Close() error
+}
+
+// CL Core OCR2 job spec RelayConfig member for Terra
+type RelayConfig struct {
 }
 
 type OCR2Spec struct {
 	ID          int32
 	IsBootstrap bool
 
-	// network data
-	TendermintURL      string // URL exposing tendermint RPC (default port is 26657)
-	CosmosURL          string // URL exposing cosmos endpoints (port is 1317, needs to be enabled in terra node config)
-	FCDNodeEndpointURL string // FCD nodes have /v1/txs/gas_prices
-	ChainID            string
-	HTTPTimeout        time.Duration
-
-	FallbackGasPrice   string
-	GasLimitMultiplier string
-
 	// on-chain data
-	ContractID string
-
-	TransmissionSigner TransmissionSigner
+	ContractID    string
+	TransmitterID string
 }
 
 type Relayer struct {
-	lggr Logger
+	lggr    Logger
+	me      MsgEnqueuer
+	tr      client.Reader
```

Comment:
Why is this called `tr`?

Reply:
terrareader - ill make it more descriptive though